browser/src/ReadData/ReadData.tsx

```diff
@@ -307,8 +325,8 @@ class ReadData extends Component<ReadDataProps, ReadDataState> {
   }
 
   loadInitialTracks() {
-    ;['exome', 'genome'].forEach((exomeOrGenome) => {
-      ;['het', 'hom', 'hemi'].forEach((category) => {
+    ;(['exome', 'genome'] as SequencingType[]).forEach((exomeOrGenome) => {
```

Comment:
if this works, it'd be more idiomatic to write `['exome', 'genome'].forEach((exomeOrGenome: SequencingType) => ...`. I _think_ TS should be able to infer that properly. Same deal in the next line.

Reply:
I would have liked that better too, but unfortunately TS complains about doing it that way, as the `forEach` is being called on an array of strings.